src/Presentation/UiBuilder.php

```diff
@@ -48,6 +49,31 @@ private function parseQuery( string $searchQuery ): Query {
 		return $this->queryStringParser->parse( $searchQuery );
 	}
 
+	/**
+	 * @return array<array<string, string>>
```

Comment:
(Not specific to this PR)

IMO these docs are pointless. Let's disable the PHPStan rule? Perhaps we can disable it selectly for `Presentation/` and `Persistence/`, where it often does not make sense to type arrays.

cc @malberts

Reply:
Is there a better way to define the types in an iterative array?
I often run into the same PHPStan rule and I am not sure what would be the best practice.

Reply:
We don't have any code that is helped by the type defined in this comment. It goes straight to the template parser. So it's wasted effort and noise.

If you look for `@return array` or `@param array` in Application/, you will find cases where the docs are useful since we call those functions.